src/core/annotation.js

```diff
@@ -617,14 +623,63 @@ class MarkupAnnotation extends Annotation {
     super(parameters);
     const dict = parameters.dict;
 
-    if (!dict.has('C')) {
-      // Fall back to the default background color.
-      this.data.color = null;
+    this.data.isMarkup = true;
+
+    if (dict.has('IT')) {
+      this.data.intent = dict.get('IT').name;
+    }
+
+    if (dict.has('IRT')) {
+      this.data.inReplyTo = dict.getRaw('IRT').toString();
+      this.data.replyType =
+      dict.has('RT') ? dict.get('RT').name : AnnotationReplyType.REPLY;
```

Comment:
This needs to be indented with two extra spaces.

Reply:
*I'm going to comment on this __once__ here, but this problem exists in many places.*

Just because a particular object exists in a `Dict`, that does *not* imply that it's actually of the desired type (i.e. `Name` here). Hence you need to check that as well, before attempting to access a potentially non-existent `name` property, since there will potentially be errors thrown otherwise.